debian/compat

```diff
@@ -0,0 +1 @@
+8
```

Comment:
Let's go with 9. 8 throws deprecation warnings on Debian and trusty already had debhelper 9. This also means adjusting debian/control to build-depend on debhelper (>= 9).